frontend/pages/register.vue

```diff
@@ -146,33 +138,32 @@ const canSubmit = computed(() => email.value && password.value && confirmPasswor
 <style lang="scss" scoped>
 @use "~/assets/css/fonts.scss";
 
-.content {
+.page {
   display: flex;
```

Comment:
like for the login, the `flex` props are not necessary here.

Reply:
Removed.